web-bluetooth/notifications.js

```diff
@@ -0,0 +1,47 @@
+var myCharacteristic;
```

Comment:
Not a good one I can find out ;)
I've fixed this at 360f5a991736426e8c2b18fbab772729a2137558